ncrystal_python/src/NCrystal/ncmat2endf.py

```diff
@@ -86,9 +86,40 @@ def _endf_roundoff(x):
         Processed array
 
     """
-    return np.array(read_fort_floats(write_fort_floats(x, {'width':11}),
+    try:
+        # TODO: temporary fix to avoid syntax warning from endf-parserpy
+        # https://github.com/IAEA-NDS/endf-parserpy/issues/10
+        import warnings
+        with warnings.catch_warnings():
+            warnings.filterwarnings("ignore",category=SyntaxWarning)
+            from endf_parserpy.interpreter.\
+                 fortran_utils import ( read_fort_floats,
+                                        write_fort_floats )
+    except ImportError:
+        raise SystemExit('Could not import endf_parserpy. Check the package '+
+                         'was correctly installed, with a version equal or '+
+                         'higher than 0.11.0.\n'+
```

Comment:
The minimal version, 0.11.0, is hardcoded in many places, and we have to keep it synchronized both internally in this file and in the dependency database.

Could we perhaps simply do away with the version check in the python code here, and instead just rely on the dependency database (`devel/dependency_database.yml`) ? That is anyway what we do for other dependencies.

Reply:
Yes, I agree. As with the imports, this script will now be part of the distribution. Makes sense.

Reply:
I will keep the endf-parserpy version in the ENDF-6 comments for V&V but remove the check there.